src/api.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
As of python 3, these lines are no longer necessary! `utf-8` is assumed by default: https://docs.python.org/3/reference/lexical_analysis.html#encoding-declarations